metabase/metabase
# Allow only single value to be selected for variable template tags

Epic https://github.com/metabase/metabase/issues/26898

How to test:
- New -> SQL Query
- `SELECT * FROM PRODUCTS WHERE CATEGORY = {{tag}}`
- Dropdown list -> Edit -> Custom list -> `Gadget`, `Gizmo`
- Click on the filter in the query editor
- Pick any value, then pick another value
- Only the last value should remain


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/28118)
